packages/reporter/src/lib/file-opener.tsx

```diff
@@ -1,43 +0,0 @@
-import { observer } from 'mobx-react'
```

Comment:
This component is no longer needed now that we delegate this workflow through graphql.